### DISCUSSION TITLE
custom Workflows?

### DISCUSSION THREAD

**Discussion by Lustwaffel** [Comfy-Org/ComfyUI-Manager#596] [General]
Dear Lt Cmdr,

Sorry, I wouldn't know how to contact you otherwise.
I want to get into art printing, have been keeping a keen eye on your youtube channel and was wondering wether you are interested in making custom workflows(realistic and managable as is in your abilities) for me(non nsfw) and what you would like to earn from each of these. 
If you have time & interest I would love to hear from you. Otherwise, no problem. 

Greetings.

**Comment by ltdrdata** [comment]
I'm sorry, my timetable is quite tight and I don't have much availability. 

Moreover, for workflow creation, I recommend looking into others on Discord ([banodoco](https://discord.gg/xkqTGBtj), [olivio](https://discord.gg/bw5HePMn)) who might be more capable in this area than I am.
The workflow I create is just a very basic one, intended as an example of how to utilize nodes.
